source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -1,10 +1,13 @@
+runOn:
+    -
+        minServerVersion: "4.1.0"
+        topology: ["replicaset", "sharded"]
```

Comment:
Correct me if I'm wrong, but this is really the only immediate need for adding `runOn`.

Reply:
that is correct